src/views/tls/tls-version/index.js

```diff
@@ -53,22 +96,22 @@ const TLSversion = () => {
         />
         <Paper square>
           <Tabs
-            value={table}
+            value={year}
             indicatorColor="primary"
-            textColor="primary"
-            onChange={(_, newValue) => setTable(newValue)}
+            onChange={(_, newValue) => setYear(newValue)}
           >
-            <Tab
-              data-testid="sample-tab"
-              value="httparchive.smaller_sample_requests"
-              label="Small sample sets"
-            />
-            <Tab
-              data-testid="10k-tab"
-              value="httparchive.sample_data.requests_desktop_10k"
-              label="10k sample set"
-            />
+            <Tab value="sample" label="Sample Data" />
```

Comment:
`data-testid` shouldn't be removed since it is for testing purposes.

You might change the tests for this component in the `__mocks__` folder next to the component to test the tab change behaviour.

Reply:
Thanks!